en/docs/administer/capturing-system-data-in-error-situations.md

```diff
@@ -40,33 +40,36 @@ inconsistencies.
     
 
 When using the tool, you have to provide the process ID (pid) of the
-Carbon instance and the \<PRODUCT\_HOME\> which is where your unzipped
+Carbon instance and the `<IS_HOME>` that is where your unzipped
 Carbon distribution files reside. The command takes the following
 format:
 
-``` java
+```shell
 sh carbondump.sh [-carbonHome path] [-pid of the carbon instance]
 ```
 
-For example,
 
-``` java
-In Linux: sh carbondump.sh -carbonHome /home/user/wso2carbon-3.0.0/ -pid 5151
+!!! example
+    ```shell tab="Linux"
+    sh carbondump.sh -carbonHome /home/user/wso2carbon-3.0.0/ -pid 5151
+    ```
+
+    ```shell tab="Windows"
+    carbondump.bat -carbonHome c:\wso2carbon-3.0.0\ -pid 5151
+    ```
 
-In Windows: carbondump.bat -carbonHome c:\wso2carbon-3.0.0\ -pid 5151
-```
 
 The tool captures the following information about the system:
 
--   Operating system information \*\* OS (kernel) version
```

Comment:
Are these forward slashes supposed to be there?

Reply:
Fixed